dojo/tools/bearer_cli/parser.py

```diff
@@ -46,6 +46,7 @@ def get_findings(self, file, test):
                     sast_source_line=bearerfinding["source"]["start"],
                     sast_source_file_path=bearerfinding["filename"],
                     vuln_id_from_tool=bearerfinding["id"],
+                    unique_id_from_tool=bearerfinding["fingerprint"],
```

Comment:
It is safe now as it is. To prevent someone(tm) from adding the `unique_id_from_tool` to the hash code config should we add a little comment here and in `settings.dist.py`? @Maffooch

Reply:
It doesn't hurt to be more verbose! @wolframite cab you please summarize your analysis of the fingerprint field in a comment where @valentijnscholten pointed out?

Reply:
Sure, no problem. I added the comment, please let me know if you're ok with the wording

Reply:
Thanks. I just realized there's one more thing... Could you update the testcase(s) to assert on the fingerprint being set into `unique_id_from_tool`?

Reply:
Huh, I don't know how I overlooked the test for bearer, I was pretty sure there wasn't one. It's updated it now. Do you want me to fixup all changes and rebase with dev?

Reply:
Also thank you @valentijnscholten and @Maffooch for guiding me along my first ticket!